packages/block-library/src/video/index.js

```diff
@@ -21,6 +21,12 @@ export const settings = {
 	description: __( 'Embed a video from your media library or upload a new one.' ),
 	icon,
 	keywords: [ __( 'movie' ) ],
+	example: {
+		attributes: {
+			src: 'https://make.wordpress.org/design/files/2019/03/block-manager.mp4',
```

Comment:
This doesn't play in Safari.

Reply:
Is there any way to have a very small auto-playing video instead? It doesn't make sense to load a video that you cannot play.

Reply:
So, I found this place for public domain video: https://www.pond5.com/stock-video-footage/1/.html#1/18447/durationlt:10,pricelt:0

This one is cute, albeit short: https://www.pond5.com/stock-footage/44576423/little-smokey-walking-cage.html

But the problem then becomes, can we hotlink it? Where should these things be hosted? Is make.wordpress.org okay?